src/disco/components/Addon/styles.scss

```diff
@@ -20,7 +20,7 @@ $addon-padding: 20px;
     color: $header-copy-font-color;
     font-size: 12px;
     margin: 0.75em 0;
-    word-break: break-all;
+    word-break: break-word;
```

Comment:
@willdurand Can we use word-wrap property to solve [0]
0:https://github.com/mozilla/addons-frontend/issues/7580#issuecomment-464088331

Reply:
ohhh, I guess yes. Wanna try it?

Reply:
Yes, Sure :)